mkdocs.yml

```diff
@@ -113,32 +113,39 @@ markdown_extensions:
 nav:
 - Home: index.md
 - Getting Started:
-  - Your first experiment: getting-started/your-first-experiment.md
-  - Automatic Routing: getting-started/first-routing.md
+  - Your first automated routing: getting-started/first-routing.md
+  - Your first performance test: getting-started/first-performance.md
+  - Your first A/B test: getting-started/first-abn.md
   - Concepts: getting-started/concepts.md
   - Install Iter8: getting-started/install.md
   - Get help: getting-started/help.md
 - Tutorials:
-  - Load test multiple HTTP endpoints: tutorials/load-test-http-multiple.md
-  - Load test gRPC: tutorials/load-test-grpc.md
-  - Load test multiple gRPC methods: tutorials/load-test-grpc-multiple.md
-  - A/B testing with Iter8 SDK: tutorials/abn/abn.md
+  - Automated routing:  tutorials/automated-routing.md
+  - Performance test HTTP:  tutorials/load-test-http.md
+  - Performance test multiple HTTP endpoints: tutorials/load-test-http-multiple.md
+  - Performance test gRPC: tutorials/load-test-grpc.md
+  - Performance test multiple gRPC methods: tutorials/load-test-grpc-multiple.md
+  - A/B testing with Iter8 SDK: tutorials/abn.md
   - Integrations:
     - Kserve:
       - Performance testing:
         - HTTP: tutorials/integrations/kserve/http.md
         - gRPC: tutorials/integrations/kserve/grpc.md
-      - Blue-Green testing: tutorials/integrations/kserve/blue-green.md
+      - Blue-green testing: tutorials/integrations/kserve/blue-green.md
       - Canary testing: tutorials/integrations/kserve/canary.md
-      - Other (Iter8 v0.7): tutorials/integrations/kserve/kserve.md
     - Kserve (modelmesh):
-      - Blue-Green: tutorials/integrations/kserve-mm/blue-green.md
+      - Blue-green: tutorials/integrations/kserve-mm/blue-green.md
```

Comment:
Rename as `Blue-green testing` and `Canary testing` to match what we have to KServe.